### PR TITLE
Change NeoFS global configuration with Makefile

### PR DESCRIPTION
**Pull request by alexvanin**
This PR provides the way to change some NeoFS global configuration values. In this example there are targets for epoch duration and max object size. Also epoch duration value is removed from inner ring config. Read [this](https://github.com/nspcc-dev/neofs-node/pull/479#issue-616125625) to find more about this decision.

To change epoch duration use new Makefile target with `val` variable.
```
$ make update.epoch_duration val=30
Changing EpochDuration configration value to 30
Enter account NNudMSGzEoktFzdYGYoNb3bzHzbmM1genF password > 
Sent invocation transaction 685462e60d6989ed9257fabb80f3992d2049cb14043d1455ee7b78c3ea4da3c1
Updating NeoFS epoch to 2
Enter account NNudMSGzEoktFzdYGYoNb3bzHzbmM1genF password > 
Sent invocation transaction 16b6a58feaf756d57463825ab04c0e170d43905aa5373c7b1076615ea30a67dc
```

Possible issues:
- it is not obvious that `make prepare.ir` requires empty password, but update config commands require password `one`.

### CODE REVIEW

File: bin/config.sh [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+. services/ir/.ir.env
+
+# NeoGo binary path.
+NEOGO="${NEOGO:-docker exec -it main_chain neo-go}"
+
+# Wallet files to change config value 
+WALLET="${WALLET:-services/chain/node-wallet.json}"
+WALLET_IMG="${WALLET_IMG:-wallets/node-wallet.json}"
+
+# NeoFS configuration record: key is a sting and value is an int
```
Comment (carpawell) [inline bin/config.sh:14]: `stRing`?

File: bin/config.sh [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+int:${VALUE} -- ${ADDR} || exit 1
+
+# Fetch current epoch value
+EPOCH=`${NEOGO} contract testinvokefunction -r \
+http://morph_chain.${LOCAL_DOMAIN}:30333 \
+${NEOFS_IR_CONTRACTS_NETMAP} \
+epoch | grep 'value' | awk -F'"' '{ print $4 }'`
+
+# Update epoch to apply new configuartion value
+echo "Updating NeoFS epoch to $((EPOCH+1))"
```
Comment (carpawell) [inline bin/config.sh:42]: is that necessary to update epoch immediately?

**Review by carpawell — CHANGES_REQUESTED** [review]

File: bin/config.sh (hunk 2 above)
Comment (alexvanin) [reply bin/config.sh:42]: Yes, otherwise you'll have to wait 5 minutes before epoch duration will be decreased to 30s.

Comment (carpawell) [reply bin/config.sh:42]: ok, just wanted to say that output confused me a little bit: like i did smth wrong

**Review by carpawell — APPROVED** [review]

**Pull request merged**